lib/tasks/populate_organisation.rake

```diff
@@ -0,0 +1,44 @@
+require 'sidekiq/api'
+#use 'rake populate_organisation:organisation' to create or update organisation details
+namespace :populate_organisation do
+  task organisation:  :environment do
+    organisation_fields_mapping = {
+      "name_en" => "name_en",
+      "name_zh_tw" => "name_zh",
+      "website" => "url",
+    }
+    #set organisation_type
+    organisation_type_name = "NGO"
+    organisation_type = OrganisationType.find_by_name_en(organisation_type_name) || OrganisationType.create(name_en: organisation_type_name ,name_zh_tw: organisation_type_name ,category_en: organisation_type_name  , category_zh_tw: organisation_type_name)
+
+    #set country
+    country_name = "China - Hong Kong (Special Administrative Region)"
+    country_code = "HKG"
+    country_region = "Asia"
+
+    #for job
+    stockit_id = StockitCountryAddJob.perform_now(country_name, country_name, country_code, country_region)
+    country = Country.find_by(stockit_id: stockit_id)|| Country.create(name_en: country_name ,name_zh_tw: country_name)
+
+    begin
+      file = Nestful.get("https://goodcitystorage.blob.core.windows.net/public/s88-orgs.json").response.body
+    rescue Exception => e
+      Airbrake.notify(e, error_class: "populate_organisation", error_message: "Organisation File Error")
+    end
+
+    # file = File.read('app/assets/organisation.json')
+    if (file.present?)
+      JSON.parse(file).each do |data|
+        organisation_fields_mapping =  organisation_fields_mapping.keep_if { |k, v| data.key? v }
+        organisation = Organisation.find_by(registration: data['org_id']) ||
+          Organisation.new(registration: data['org_id'], organisation_type: organisation_type, country: country)
+          organisation_fields_mapping.each do |organisation_column, data_key|
```

Comment:
This is good. Can you just fix the indentation slightly so it's easier to read? Thanks